app/javascript/packages/document-capture/context/upload.jsx

```diff
@@ -5,6 +5,15 @@ const UploadContext = createContext(defaultUpload);
 
 /** @typedef {import('react').ReactNode} ReactNode */
 
+/**
+ * Upload field error, after normalized to error instance.
+ *
+ * @typedef UploadFieldError
+ *
+ * @prop {'front'|'back'|'selfie'|'network'} field Field name.
```

Comment:
Slow on the draw here, but we also need to be aware of the `results` prop which can pop up here: https://github.com/18F/identity-idp/blob/0b6cc9389c0e88ad412f09f1734669c832429f0a/app/services/doc_auth/acuant/responses/get_results_response.rb#L59

Reply:
Good point! Luckily the code will "fall back" to a top-level nav if we get an unknown field name